aws/delta-checkpoint/Cargo.toml

```diff
@@ -23,12 +23,17 @@ tokio-stream = { version = "0", features = ["fs"] }
 [dependencies.deltalake]
 path = "../../rust"
 version = "0"
-features = ["s3-rustls"]
 
 [dev-dependencies]
-rusoto_core = { version = "0.48", default-features = false, features = ["rustls"] }
+rusoto_core = { version = "0.48", default-features = false }
 rusoto_credential = "0.48"
-rusoto_s3 = { version = "0.48", default-features = false, features = ["rustls"] }
+rusoto_s3 = { version = "0.48", default-features = false }
+
+[features]
+default = ["native-tls"]
```

Comment:
for the checkpoint lambda, we should probably set rustls as the default feature because it's hard to get openssl working in the lambda runtime.

Reply:
If we want to be able to run `cargo test` at the root, we need all packages to either default to `native-tls` or `rustls`. I can add a note in the readme that we recommend rustls for this package.

Reply:
Ha I see. What if we default all to rustls? I think this might be what's preferred for our python releases as well?

Reply:
Or maybe I'll just make `rustls` default.